apps/frontend/assets/src/omg-2fa-modal/Disable2FaModal.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { withRouter } from 'react-router-dom'
+
+import Modal from '../omg-modal'
+import { disable2Fa } from '../omg-2fa/action'
+import { Input, Button, Icon } from '../omg-uikit'
+
+const Disable2FaModalContainer = styled.div`
+  padding: 50px;
+  text-align: center;
+  width: 450px;
+  position: relative;
+  i {
+    position: absolute;
+    top: 15px;
+    right: 15px;
+    cursor: pointer;
+  }
+
+  input {
+    text-align: center;
+    font-size: 18px;
+  }
+
+  button {
+    margin-top: 30px;
+  }
+  h4 {
+    margin-bottom: 10px;
+  }
+  .backup-container {
+    text-align: center;
+    p {
+      margin: 20px 0;
+    }
+  }
+  .backup-item {
+    padding: 5px;
+    display: inline-block;
+    width: 110px;
+  }
+`
+
+function DisableTwoFaModal ({ open, onRequestClose, history }) {
+  const dispatch = useDispatch()
+  const [passcode, setPasscode] = useState('')
+  const [submitStatus, setSubmitStatus] = useState('DEFAULT')
+  const [errorText, setErrorText] = useState()
+  const afterClose = () => {
+    setPasscode('')
+    setSubmitStatus('DEFAULT')
+  }
+
+  const onSubmit = async e => {
+    e.preventDefault()
+    setSubmitStatus('LOADING')
+    const result = await disable2Fa(passcode)(dispatch)
+    if (result.data) {
+      setSubmitStatus('SUCCESS')
+      onRequestClose()
+      history.push('/login')
+    } else {
+      setErrorText(result.error.description)
+      setSubmitStatus('FAILED')
+    }
+  }
+
+  const renderDisableSection = () => {
+    return (
+      <form onSubmit={onSubmit}>
+        <h4>Disable Your Two Factor Authentication</h4>
+        <Input
+          value={passcode}
+          onChange={e => setPasscode(e.target.value)}
+          normalPlaceholder='passcode...'
```

Comment:
maybe `2FA Token` is more clear? maybe people will think this is asking for their password

Reply:
I'm using the same name as what backend use, can change the placeholder